packages/web/src/components/ui/Package/Package.tsx

```diff
@@ -19,19 +27,82 @@ export type Props = {
 };
 
 export default function Package({ className, variant = 'grid', pkg }: Props) {
+  const repositoryUrl = pkg.registryMetadata?.repositoryUrl;
+  const homepageUrl = pkg.registryMetadata?.homepageUrl;
+  const isOutdated =
+    pkg.registryMetadata && semver.gtr(pkg.registryMetadata.latestVersion, pkg.packageVersionRange);
 
   return (
     <div className={clsx(styles.container, styles[variant], className)}>
-      <div className={styles.main}>
-        <div className={styles.name}>{pkg.packageName}</div>
-        <div className={styles.version}>{pkg.packageVersionRange}</div>
+      <div className={styles.registryMeta}>
+        {isOutdated && <div className={styles.badge}>Outdated</div>}
+        <div className={styles.externalLinks}>
+          {repositoryUrl && (
+            <a
+              href={repositoryUrl}
+              target='_blank'
+              rel='noopener noreferrer'
+              className={styles.externalLink}
+            >
+              <Github />
+            </a>
+          )}{' '}
+          {homepageUrl && homepageUrl !== repositoryUrl && (
+            <a
+              href={homepageUrl}
+              target='_blank'
+              rel='noopener noreferrer'
+              className={styles.externalLink}
+            >
+              <External />
+            </a>
+          )}
+        </div>
       </div>
+      <a
+        className={styles.name}
+        href={`https://www.npmjs.com/package/${pkg.packageName}`}
+        target='_blank'
+        rel='noopener noreferrer'
+        aria-label={pkg.packageName}
+        // Browser won't break a line for '/' symbol, so we add the <wbr> specificaly
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: pkg.packageName.replace('/', '/<wbr>') }}
+      />
       <div className={styles.meta}>
+        <div className={styles.version}>{toReadableVersion(pkg.packageVersionRange)}</div>
         {!!pkg.packageMetadata?.approximateByteSize && (
-          <span className={styles.size}>{pkg.packageMetadata.approximateByteSize}B</span>
+          <span className={styles.size}>
+            {toReadableSize(pkg.packageMetadata.approximateByteSize)}
+          </span>
         )}
-        {/* <span className={styles.percent}>23%</span> */}
       </div>
     </div>
   );
 }
+
+const byteUnits = ['B', 'KB', 'MB'];
+function toReadableSize(size: number) {
+  let byteUnitIndex = 0;
+  let sizeInUnits = size;
+
+  while (sizeInUnits > 1024) {
+    sizeInUnits /= 1024;
+    byteUnitIndex += 1;
+  }
+
+  let readableSize = sizeInUnits.toFixed(1);
+  if (readableSize.slice(-2) === '.0') {
```

Comment:
`.endsWith()`?

Reply:
There's also a more common but less readable approach of `parseFloat(float.toFixed(x))` which does the trick

Reply:
Fixed